packages/notte-core/src/notte_core/profiling.py

```
@@ -0,0 +1,894 @@
+
+        print("OpenTelemetry Profiling Results:")
+        print("-" * 80)
+        print(f"{'Operation':<40} {'Start':<10} {'End':<10} {'Duration':<12}")
+        print("-" * 80)
+
+        for item in flamegraph_data:
+            indent = "  " * item["depth"]
+            name = f"{indent}{item['name']}"
+            print(f"{name:<40} {item['start_time']:<10.6f} {item['end_time']:<10.6f} {item['duration']:<12.6f}")
+
+    def generate_flamegraph_svg(
```

Pylint (3.3.7)

[refactor] Too many local variables (52/15)

(R0914)